dumper.py

```diff
@@ -1,33 +1,66 @@
-#!/usr/local/bin/python3
-import logging
+#!/usr/bin/env python3
 import argparse
-from redmine.redmine_dumper import RedmineDumper
+import logging
+import logging.config
+import yaml
 
+from redmine.redmine_dumper import RedmineDumper
+from bsc.bugzilla_suse import bzmain
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument(
-        '--source', help="Source from where ticket will be dumped. Possible values: (redmine|bugzilla). \
-            Default: redmine", default='redmine')
     parser.add_argument('--database', help="SQLite file path", required=True)
-    parser.add_argument('--project', help="Project name")
     parser.add_argument('-v', action='store_true', help='increase output verbosity')
+    subparsers = parser.add_subparsers(dest='source', required=True)
+    redmine_parser = subparsers.add_parser('redmine', help='redmine help')
+    redmine_parser.add_argument('--project', help="Project name")
+    
+    bsc_parser = subparsers.add_parser('bugzilla', help='bugzilla help')
```

Comment:
As a general suggestion the "bugzilla" parser should probably not refer to "bsc" anywhere, and be instance-agnostic like the "redmine" parser?

Reply:
Had a brief chat with Ioannis. Maybe both "redmine" and "bugzilla" could live in a "trackers" module, which would also avoid clobbering the top-level of the repo.